submissions/Nemooochka/movie-seats/index.html

```diff
@@ -0,0 +1,301 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="style.css">
+    <title>Movie seats</title>
+</head>
+<body>
+    <header>
+        <div class="container">
+            <h2>
+                Your favourite movie
+            </h2>
+        </div>
+    </header>
+    <main>
+        <div class="container">
+            <section class="cinema">
+                <div class="cinema-tickets_types">
+                    <div class="cinema-tickets_type type-good">
+                        <span class="rectangle"></span>
+                        <span class="type">GOOD -</span>
```

Comment:
When I change text will it be uppercase? Use CSS for styling text.
Avoid setting styles in markup.